pkg/signup/service/signup_service.go

```diff
@@ -284,6 +284,7 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 			if member.ClusterName == mur.Status.UserAccounts[0].Cluster.Name {
 				signupResponse.ConsoleURL = member.MemberStatus.Routes.ConsoleURL
 				signupResponse.CheDashboardURL = member.MemberStatus.Routes.CheDashboardURL
+				signupResponse.ApiEndpoint = mur.Status.UserAccounts[0].Cluster.APIEndpoint
```

Comment:
Aren't you moving the apiendpoint to the toolchainstatus resource?
```suggestion
				signupResponse.ApiEndpoint = member.APIEndpoint
```

Reply:
You're right, i should change it to member.apiEndpoint. Never came back to changing this logic after moving it to member

Reply:
my apologies, i could have used the commit suggestion instead